RJL.HW4.OOP.Classes.Task2/Agregat.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RJL.HW4.OOP.Classes.Task2
+{
+    class Agregat
+    {
+        public int GeneralCountDetails { get; private set;}
+        public int CurrentAssembledDetails { get; set;}
```

Comment:
You allow to assign for `CurrentAssembledDetails ` -100000 or 20000000 when `GeneralCountDetails` is 15 for example